core/drivers/sub.mk

```diff
@@ -5,6 +5,8 @@ srcs-$(CFG_TZC380) += tzc380.c
 srcs-$(CFG_GIC) += gic.c
 srcs-$(CFG_PL061) += pl061_gpio.c
 srcs-$(CFG_PL022) += pl022_spi.c
+srcs-$(CFG_TPM2) += tpm2.c
+srcs-$(CFG_TPM2) += tpm2_mmio.c
```

Comment:
This should go into the commit "tpm2: add MMIO platform driver"

Reply:
deserves a specific `CFG_TPM2_MMIO`?